js/classes/game_engine.js

```diff
@@ -123,14 +123,14 @@ class GameEngine {
      */
     startInput() {
 
-        CANVAS.addEventListener("mouseDown", (e) => {
-            this.user.firing = true;
-            this.user.aiming = false;
+        CANVAS.addEventListener("mousedown", (e) => {
+            this.user.firing = false;
+            this.user.aiming = true;
         });
 
-        CANVAS.addEventListener("mouseUp", (e) => {
-            this.user.aiming = true;
-            this.user.firing = false;
+        CANVAS.addEventListener("mouseup", (e) => {
+            this.user.aiming = false;
```

Comment:
It's weird to me that everything was reversed prior to your changes.

Reply:
The way you had it made intuitive sense - you were setting firing to true when the mouse was down and setting aiming to true when the mouse was up. It's just that we want our slingshot to work in the opposite way.